integration/image/main_test.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/docker/docker/internal/test/environment"
 )
 
+const (
+	privateRegistryURL = "127.0.0.1:5000"
```

Comment:
What's this for?

Reply:
remove this? I think it's unused.

Reply:
Oups.. forgot to commit+push :sweat_smile: